build.gradle

```diff
@@ -2,6 +2,8 @@ plugins {
     id 'java'
     id 'org.jetbrains.intellij' version '0.4.18'
     id 'io.franzbecker.gradle-lombok' version '3.3.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+    id "org.jlleitschuh.gradle.ktlint" version "8.2.0"
```

Comment:
I figured I would also throw in a linter for Kotlin. I like the indents set to 2, but that's just me.

Reply:
We could do `2` indents on Kotlin, but we should retain `4` indents on Java. Or is it worth it to reduce the indents to existing Java files while it is still small? As this would make it consistent with DX?

Reply:
:thinking: Hmm, yes it would be nice for things to be consistent. I think for right now I'll bump it up to 4. Perhaps another PR can be set up just for lint changes?